UnityProject/Assets/Scripts/Managers/MatrixManager/MatrixManager.cs

```diff
@@ -79,6 +68,18 @@ private IEnumerator WaitForLoad()
 		EventManager.Broadcast(EVENT.MatrixManagerInit);
 	}
 
+	private void Awake()
+	{
+		if (matrixManager == null)
+		{
+			matrixManager = this;
+		}
+		else
+		{
+			Destroy(this);
```

Comment:
could this be the thing making tests go ape?

Reply:
yup, cus of execute in edit mode, trying to find a way to fix

Reply:
Added a check which fixes this